Remove the promise of fixing (#251)
memgraph/docs: docs/changelog.md

### Known Bugs

> * Loading streams created by older versions cause Memgraph to crash. The only

Reviewer: Shouldn't we also mention something as part of breaking changes? We broke the persisted streams.

It is hard to differentiate the breaking change (we cannot load old streams) and the bug (while memgraph tries to load old streams, it crashes), but I think we should separate them.

Author: We could separate them, but we decided not to fix the general problem, only the unfortunate side effect which is crash instead of silent warning and skip. So I would push everything to Breaking changes.

* Now supporting Bolt protocol version 4.3.
  [#226](https://github.com/memgraph/memgraph/pull/226)
* Streams support for retrying conflicting transactions. When a message is processed from
  a certain stream source, a query is executed as a part of the transaction. If that transaction

>   fails because of other conflicting transaction, the transaction is retried set number of times.

Reviewer 2: ```suggestion
  fails because of other conflicting transactions, the transaction is retried set number of times.
```

>   fails because of other conflicting transactions, the transaction is retried set number of times.

Reviewer 3: ```suggestion
  fails because of other conflicting transactions, it is retried a configurable number of times.
```

>   Number of retries and interval between each retry can be controled with configs

Author: ```suggestion
  The number of retries and interval between each retry can be controlled with configs
```
